cider/exec/nextgen/jitlib/base/JITValueOperations.h

```diff
@@ -57,6 +78,94 @@ JITValuePointer operator+(T lh, JITValue& rh) {
   return rh + lh;
 }
 
+JITValuePointer operator-(JITValue& lh, JITValue& rh) {
+  return lh.sub(rh);
+}
+
+template <typename T, typename = std::enable_if_t<is_jitvalue_convertable_v<T>>>
```

Comment:
`is_arithmetic_v<T>` maybe better? why wo need `operator-` for bool?

Reply:
Hmm, it may be useless, but not forbidden😂